truffle/src/main/java/org/jruby/truffle/nodes/ext/BigDecimalNodes.java

```diff
@@ -178,175 +217,327 @@ public static boolean isNan(RubyBasicObject value) {
             return getBigDecimalType(value) == Type.NAN;
         }
 
+        private void setupCreateBigDecimal() {
+            if (createBigDecimal == null)
+                createBigDecimal = insert(CreateBigDecimalNodeFactory.create(getContext(), getSourceSection(), null, null));
```

Comment:
I'll let @chrisseaton and @eregon weigh in, but I I think everything we've written up until now (that wasn't directly ported from elsewhere) always uses curly braces.  I'm personally not a fan of relying on indentation in a language that doesn't enforce it, and as such would vote in favor of always using braces.